lib/features/modeling/behavior/DeleteLaneBehavior.js

```diff
@@ -106,7 +150,9 @@ export default function DeleteLaneBehavior(eventBus, spaceTool) {
       return;
     }
 
-    compensateLaneDelete(shape, oldParent);
+    var isHorizontalLane = isHorizontal(shape);
+
+    compensateLaneDelete(shape, oldParent, isHorizontalLane);
```

Comment:
I'd rather move the `isHorizontal` call to the inside of `compensateLaneDelete`. There's no case where we'd pass `isHorizontalLane=false` for horizontal lanes to this function.

Reply:
I will do that as a quick change.